connectors/spark/src/main/scala/io/unitycatalog/spark/ApiClientFactory.java

```diff
@@ -5,13 +5,18 @@
 import java.net.URI;
 
 public class ApiClientFactory {
+
+  public static final String BASE_PATH = "/api/2.1/unity-catalog";
+
   private ApiClientFactory() {}
 
   public static ApiClient createApiClient(URI url, String token) {
+    String basePath = url.getPath() + BASE_PATH;
```

Comment:
Why do we need to `concat` the `url.getPath()` and `BASE_PATH` to set the `basePath` for the ApiClient ?

Reply:
( since we've already set the host and port for the `apiClient` ).